README.md

```diff
@@ -0,0 +1,134 @@
+This is a client implementation of the Apache Camel Language Server Protocol for Emacs
+
+# Camel LSP client for Emacs
+
+Code completion for Camel LS with XML LS
+
+![Demo](images/camell.gif)
+
+NOTE: Completion is with Ctrl+Alt+i 
+
+## Prerequisites
+
+
+In a `~/.emacs.d/init.el` file:
+
+```
```

Comment:
You can add the language name here so that the code block benefits from syntax highlighting. see https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting